### PR TITLE
edit landing page, kill debug, change some copy

### PR DESCRIPTION
**Pull request by Guaris**
Points to pricing page for capabilities
Edits the capabilities table
Removed debug page
Removes mention of the debug page
Changes some copy to emphasize the test + design relationship.

### CODE REVIEW

File: app/_landing_pages/insomnia.yaml [hunk 1]
```diff
@@ -144,45 +129,33 @@ rows:
               blocks:
                 - type: text
                   text: | 
-                    See the full [feature comparison &rarr;](/insomnia/feature-comparison)
```
Comment (lmilan) [inline app/_landing_pages/insomnia.yaml:147]: We should also delete the feature comparison page

File: app/_landing_pages/insomnia.yaml [hunk 2]
```diff
@@ -47,27 +46,18 @@ rows:
...
-            icon: /assets/icons/insomnia/bug.svg
-            cta:
-              text: Learn more
-              url: /insomnia/debug
-              align: end
-      - blocks:
-        - type: card
-          config:
-            title: Insomnia Plugin Hub
+            title: Plugins
```
Comment (lmilan) [inline app/_landing_pages/insomnia.yaml:60]: Do we want to go straight to the plugin hub, or to the plugins landing page?

File: app/_landing_pages/insomnia.yaml [hunk 3]
```diff
@@ -91,37 +81,32 @@ rows:
...
                   key: ee
               features:
-                - title: Local vault for local storage of collections, design specs, and every other resource
+                - title: Local storage
                   oss: true
                   individual: true
                   team: true
                   ee: true
-                - title: Debug APIs using the most popular protocols and formats
+                - title: Access to API testing suite
```
Comment (lmilan) [inline app/_landing_pages/insomnia.yaml:89]: I'm wondering if this makes it sound like it's an existing test suite, and not a feature to create test suites?

File: app/_landing_pages/insomnia.yaml (hunk 2 above)
Comment (Guaris) [reply app/_landing_pages/insomnia.yaml:60]: Plugins landing page I think.

File: app/_landing_pages/insomnia.yaml (hunk 3 above)
Comment (Guaris) [reply app/_landing_pages/insomnia.yaml:89]: How about "API testing tools"

File: app/_landing_pages/insomnia.yaml (hunk 1 above)
Comment (Guaris) [reply app/_landing_pages/insomnia.yaml:147]: done

**Pull request merged**